### DISCUSSION TITLE
Can We Make A Project Site On GitHub? I Will Manage It.

### DISCUSSION THREAD

**Discussion by ghost** [Slow-Scallion4183/Adventure-Colussus#33] [General]
Cuz We Cant Make Our Players Download Our Game From GitHub For Longer. This Place Is Only For Coding The Game. Instead, We Can Make A Site On "pages.github.com". Ill Try To Do It From Scratch. I Know Basic HTML And  I Have Some Programming Experience In Javascript And HTML.

By The Way Who The Heck Is Lareya In Watch List??

**Comment by Slow-Scallion4183** [comment]
Yeah sure the site sounds cool  :D Go ahead!. Also i have no idea who Lareya is or even that we ahve a watch list

**Reply by Slow-Scallion4183** [reply]
oh its probably just someone who is interested in the project

**Reply by ghost** [reply]
oh ok. but she didnt fork our project lol. kinda sus.

**Reply by ghost** [reply]
Code:

```<DOCTYPE html>
<title>Adventure Colussus</title>
<h1 style="color:black;">Adventure Colussus:</h1>

<body style="background-color:#708090;">

<p>Adventure Colussus Is A Text-Based 2D RPG Game Made By Some Python Developers Who Spent Their Time On This Project. The Game And This Site Is Currently Under Dev And If You Want To Join Us Just Contribute Us At The Link Given Below And Make Pull Requests. We Will Check If You Are Capable Of Developing In Our Project.</p>
 
 <h2>Devs:</h2>
  <p>---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------</p>
<p>1) Slow-Scallion4183 - Project Leader</p>
<p>2) SkepticalPotato2k - Website Manager</p>
<p>3) wireboy5 - Lead Dev</p>
<p>4) robotlightsyou - Dev </p>
<p>5) ssguitarist - Dev</p>


 
<h2>Project Links:</h2>
<p>---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------</p>
<link><link>
<a href="https://github.com/Slow-Scallion4183/Adventure-Colussus">1) Official Project GitHub</a>```

**Reply by ghost** [reply]
https://skepticalpotato2k.github.io/adventure-colussus.github.io/

**Reply by Slow-Scallion4183** [reply]
looks good! i sent you a discord message aswell so u might want to check that out

**Reply by wireboy5** [reply]
You can actually create a branch for GitHub Pages on this repo. Just call it pages, and then go into settings and select the branch under GItHub Pages under Options.

**Reply by Slow-Scallion4183** [reply]
oh i didnt know that. thanks for the info :D

**Reply by ghost** [reply]
> You can actually create a branch for GitHub Pages on this repo. Just call it pages, and then go into settings and select the branch under GItHub Pages under Options.

oh ok
